app/models/apple_credential.rb

```diff
@@ -0,0 +1,18 @@
+class AppleCredential < ActiveRecord::Base
+  belongs_to :podcast, -> { with_deleted }
+
+  validates_associated :podcast, if: -> { prx_account_uri.blank? }
+  validates_presence_of :prx_account_uri, if: -> { podcast.blank? }
```

Comment:
yeah, this seems right - we only associate accounts at the show/podcast level otherwise, and this does seem like the same level, rather than setting up credentials per show